nidx/nidx_relation/src/reader.rs

```diff
@@ -320,29 +261,52 @@ impl RelationsReaderService {
                             break;
                         }
                         let start = end - len;
-                        self.add_fuzzy_prefix_query(&mut source_prefix_q, &mut target_prefix_q, &words[start..end]);
+                        let prefix = &words[start..end];
+
+                        let normalized_prefix = schema::normalize_words(prefix.iter().copied());
+                        prefix_nodes_q.push(Node {
+                            value: Some(Term::Fuzzy(FuzzyTerm {
+                                value: normalized_prefix,
+                                fuzzy_distance: FUZZY_DISTANCE,
+                                // BUG: this should be true if we want prefix search
```

Comment:
Why not just set to true and remove the comment?

Reply:
I also had to fix some tests and I prefer to do it in a separate small PR (easier to understand, easier to revert)